_posts/2020-09-01-weather-prediction/weather_prediction_deep_learning.Rmd

```diff
@@ -0,0 +1,693 @@
+range of about six hours) over medium-range (three to five days) and sub-seasonal (weekly/monthly) to climate forecasts
+(concerned with years and decades). Medium-range forecasts especially are extremely important in acute disaster prevention.
+
+This post will show how deep learning (DL) methods can be used to generate atmospheric forecasts, using a newly published
+[benchmark dataset](https://github.com/pangeo-data/WeatherBench)[@weatherbench]. Future posts may refine the model used here
+and/or discuss the role of DL ("AI") in mitigating climate change -- and its implications -- more globally. And coming back to
+Bertolt Brecht: While this text may be about trees, climate change should not remain the only societal topic addressed on this
+blog.
+
+That said, let's put the current endeavor in context. In a way, we have here the usual *dejà vu* of using DL as a
+black-box-like, magic instrument on a task where *human knowledge* used to be required. Of course, this characterization is
+overly dichotomizing; many choices are to be made in creating DL models, and performance is necessarily constrained by
+available algorithms -- which may, or may not, fit the domain to be modeled to a sufficient degree.
+
+If you've started learning about image recognition rather recently, you may well have been using DL methods from the outset,
+and not have heard much about the rich set of feature engineering methods developed in pre-DL image recognition. In the
+context of atmospheric prediction, then, let's begin by asking: How in the world did they do that *before*?
+
+## Numerical weather prediction in a nutshell
+
+It is not like machine learning and/or statistics are not already used in numerical weather prediction -- on the contrary. For
+example, every model has to start from somewhere; but raw observations are not suited to direct use as initial conditions.
+Instead, they have to be assimilated to the four-dimensional *grid*[^2] over which model computations are performed. At the
+other end, namely, model output, statistical post-processing is used to refine the predictions. And very importantly, ensemble
+forecasts are employed to determine uncertainty.
+
+[^2]: Four, because in addition to three spatial dimensions, there is the time dimension.
+
+That said, the model *core*, the part that extrapolates into the future atmospheric conditions observed today, is based on a
+set of differential[^3] equations, the so-called [primitive equations](https://en.wikipedia.org/wiki/Primitive_equations),
+that are due to the conservation laws of [momentum](https://en.wikipedia.org/wiki/Conservation_of_momentum),
+[energy](https://en.wikipedia.org/wiki/Conservation_of_energy), and
+[mass](https://en.wikipedia.org/wiki/Conservation_of_mass). These differential equations cannot be solved analytically;
+rather, they have to be solved numerically, and that on a grid of resolution as high as possible. In that light, even deep
```

Comment:
```suggestion
rather, they have to be solved numerically on a grid of resolution as high as possible. In that light, even deep
```

Reply:
If my edit here doesn't make sense, then I don't understand the last part of the sentence 😬